settings.toml

```diff
@@ -0,0 +1,8 @@
+[default.service]
+host = "127.0.0.1"
+port = 9191
+max_download_size = 1000000000
+n_download_max_tries = 10
+download_retry_sleep = 0.5
+sentry_url = "https://63ae106793010d836c74830fa75b300c@o264756.ingest.sentry.io/4506186624335872"
+ontology_path = "https://odahub.io/ontology/ontology.ttl"
```

Comment:
These configs belong not only to service module. Probably better to have some separate blocks, e.g. [global] and [service] ?

Reply:
so, basically only `host` an `port` are specific for the service?